### ISSUE TITLE
Default location of elasticsearch?

### ISSUE THREAD

**Issue by JonesH** [codeformuenster/muenster-info-app#16]
The current configuration assumes usage of the live backend. In order to change this, 2 files (package.json+src/DataContext.js) need adjusting. Can this be done through environment variables?

**Comment by ubergesundheit** [comment]
yes. Check out: https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env

Create a file called `.env` and add the url of your local elasticsearch like this:

    REACT_APP_ELASTIC_SEARCH_URL=http://elasticsearch:9200/msinfohub

Create another file called `.env.production` with the live url

    REACT_APP_ELASTIC_SEARCH_URL=https://api.muenster.jetzt/msinfohub-events

Edit `src/DataContext.js` and replace the URL in line 66 with `process.env.REACT_APP_ELASTIC_SEARCH_URL` so that it looks like this:

            `${process.env.REACT_APP_ELASTIC_SEARCH_URL}/_search?size=2000`,

**Comment by ubergesundheit** [comment]
#18
